package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "udata-gouvfr",
-  "version": "0.1.0",
+  "version": "2.1.1",
```

Comment:
don't change that, we are not using it, we don't want to maintain it (and it might break stuff for all we know).

Reply:
oki I'll reverse that